src/ui/TextField.cpp

```diff
@@ -147,7 +159,7 @@ QColor
 TextField::underlineColor() const
 {
         if (!underline_color_.isValid()) {
-                if (hasAcceptableInput() || !isModified())
+                if (TextField::isValid() || !isModified())
```

Comment:
```suggestion
                if ((hasAcceptableInput() && isValid()) || !isModified())
```

Otherwise this breaks the registration window